truffle/src/main/java/org/jruby/truffle/nodes/core/SymbolNodes.java

```diff
@@ -241,7 +241,28 @@ public SwapcaseNode(SwapcaseNode prev) {
         public RubySymbol swapcase(RubySymbol symbol) {
             notDesignedForCompilation();
 
-            ByteList byteList = StringNodes.StringNodesHelper.swapcase(symbol.toRubyString());
+            final ByteList byteList = StringNodes.StringNodesHelper.swapcase(symbol.toRubyString());
```

Comment:
Converting to a `RubyString` isn't ideal - it would be good if we could just use the `ByteList` directly. When we are using the JIT compiler the `RubyString` wrapper object would not be allocated, but in the interpreter it's a little bit of overhead.